package.json

```diff
@@ -16,19 +16,30 @@
     "@types/cookie-parser": "^1.4.6",
     "@types/cors": "^2.8.17",
     "@types/express": "^4.17.21",
+    "@types/jest": "^29.5.12",
     "@types/lodash": "^4.14.202",
     "@types/mongoose": "^5.11.97",
+    "axios-cookiejar-support": "^5.0.0",
     "body-parser": "^1.20.2",
     "compression": "^1.7.4",
     "cookie-parser": "^1.4.6",
     "cors": "^2.8.5",
     "dotenv": "^16.4.2",
     "express": "^4.18.2",
+    "jest": "^29.7.0",
     "lodash": "^4.17.21",
     "mongodb": "^6.3.0",
     "mongoose": "^8.1.1",
     "nodemon": "^3.0.3",
+    "tough-cookie": "^4.1.3",
+    "ts-jest": "^29.1.2",
```

Comment:
the ts-jest package should generally be listed as a development dependency in your project.

Reply:
This as well